bio_hansel/main.py

```diff
@@ -140,6 +147,19 @@ def main():
         subtyping_params.min_ambiguous_tiles = args.min_ambiguous_tiles
     if args.max_intermediate_tiles:
         subtyping_params.max_perc_intermediate_tiles = args.max_intermediate_tiles
+    if args.min_kmer_freq:
+        if str(args.min_kmer_freq).lower() == "auto":
+            subtyping_params.calc_min_kmer_freq = True
+        else:
+            subtyping_params.min_kmer_freq = int(args.min_kmer_freq)
```

Comment:
Wrap in `try/except/finally` for safety and error handling. Maybe proceed with analysis despite user not providing a valid integer value, but warn them that you're using the default?

Reply:
That's a good solution in case the user puts in random garbage.